### ISSUE TITLE
Documentation Link from IDE Plugin Window

### ISSUE THREAD

**Issue by rrrix** [jimkyndemeyer/js-graphql-intellij-plugin#333]
## Version and Environment Details
__Operation system__: MacOS Catalina

__IDE name and version__: WebStorm 2020.1.1

__Plugin version__: 2.4.0

## Expected Behaviour

Link to documentation should _obvious_ and readily available from the IDE Plugin "details" view/screen, or at least somewhere at the top of the page at https://plugins.jetbrains.com/plugin/8097-js-graphql:

![IDE Plugins: JS GraphQL Plugin](https://user-images.githubusercontent.com/3194704/81445384-9e15b500-912d-11ea-824a-c36fa6224957.png)
  
## Actual Behaviour

In order to find the documentation a user must:

- Click the "Plugin Homepage" link
- Scroll past "What's New"
- Scroll past "Ratings & Reviews"
- Be looking for a link called **Documentation**

## Background

I've been using this plugin for quite a while now and could honestly never figure out why my "graphql-config" - per the documentation on [graphql-config.com](https://graphql-config.com/graphql-config) was never working. 

Finally digging through this repo's Issues, I found [#296](https://github.com/jimkyndemeyer/js-graphql-intellij-plugin/issues/296) and their "legacy" documentation. Which then I finally realized I should click the link at the top of the repo to get to this plugin's ACTUAL documentation...  

Long story short, I didn't even know you had documentation because it's very hard to find. 

Thank you for the excellent plugin :)
